File: spring-cloud-starter/src/main/java/com/jmsoftware/maf/springcloudstarter/quartz/constant/QuartzJobStatus.java
package com.jmsoftware.maf.springcloudstarter.quartz.constant;

import com.jmsoftware.maf.common.enumeration.ValueDescriptionBaseEnum;
import lombok.Getter;
import lombok.ToString;

/**
 * Description: QuartzJobStatus, change description here.
 *
 * @author Johnny Miller (锺俊), email: johnnysviva@outlook.com, date: 9/24/2021 10:11 AM
 **/
@Getter
@ToString
public enum QuartzJobStatus implements ValueDescriptionBaseEnum<Byte> {
    /**
     * Normal
     */
    NORMAL((byte) 0),
    /**
     * Paused
     */
    PAUSE((byte) 1),
    ;

    private final Byte value;

    QuartzJobStatus(Byte value) {
        this.value = value;
    }

    @Override
    public String getDescription() {
        return null;
    }
}

File: spring-cloud-starter/src/main/java/com/jmsoftware/maf/springcloudstarter/quartz/constant/QuartzJobStatus.kt
package com.jmsoftware.maf.springcloudstarter.quartz.constant

/**
 * # QuartzJobStatus
 *
 * Description: QuartzJobStatus, change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/14/22 10:32 PM
 */
enum class QuartzJobStatus(val value: Byte) {
    /**
     * Normal
     */
    NORMAL(0.toByte()),

    /**
     * Paused
     */
    PAUSE(1.toByte());
}
